samples/sample_full_pbr.cpp

```diff
@@ -281,9 +281,7 @@ static void setup(Engine* engine, View* view, Scene* scene) {
             vec2 uvDy = dFdy(uv0);
 
             mat3 tangentFromWorld = transpose(getWorldTangentFrame());
-            vec3 tangentCameraPosition = tangentFromWorld * getWorldCameraPosition();
-            vec3 tangentFragPosition = tangentFromWorld * getWorldPosition();
-            vec3 v = normalize(tangentCameraPosition - tangentFragPosition);
+            vec3 v = tangentFromWorld * getWorldViewVector();
```

Comment:
**For future reference**:
here `tangentFromWorld` is [the usual TBN matrix](https://github.com/shapr3d/filament/blob/shapr3d_master/shaders/src/shading_parameters.fs#L31) containing only rotation and mirroring. Thus it is safe to use it to transform direction vectors. It keeps the vector's length, so the normalizing at the end is not needed (because `getWorldViewVector()` is already normalized)